src/orange/detail/UnitInserter.cc

```diff
@@ -286,12 +296,26 @@ VolumeRecord UnitInserter::insert_volume(const SurfacesRecord& surf_record,
         max_intersections += get_num_intersections(sid);
     }
 
+    auto input_logic = make_span(v.logic);
+    if (v.flags & VolumeRecord::Flags::implicit_cell)
```

Comment:
Do you think it's worth adding a quick method on `VolumeInput` that returns `flags & VolumeRecord::Flags::implicit_cell`? I've seen that pattern a few times now.

Also, I noted that sometimes you have `VolumeRecord::implicit_cell` and other times `VolumeRecord::Flags::implicit_cell`. Probably best to consistently use one.

Reply:
Also I wrote `cell` instead of `volume` which Elliott didn't notice ;)

Reply:
I think I'll make `flags` an implementation detail: `VolumeView` is "high level" so I'll just add some `bool` accessors.